_studio/mfx_lib/shared/src/libmfxsw_pak.cpp

```diff
@@ -38,8 +38,10 @@
 template<>
 VideoPAK* _mfxSession::Create<VideoPAK>(mfxVideoParam& par)
 {
-    VideoPAK* pPAK = nullptr;
+#if defined(MFX_VA_LINUX) && defined(MFX_ENABLE_H264_VIDEO_ENCODE_HW) && defined(MFX_ENABLE_H264_VIDEO_FEI_PAK)
```

Comment:
I suggest to directly use m_pCore.get() below, i.e.:
`pPAK = (VideoPAK*) new VideoPAK_PAK(m_pCore.get(), &mfxRes);`
Less macros the better...